README.md

```diff
@@ -1,5 +1,5 @@
 # translate
-A [maubot](https://github.com/maubot/maubot) to translate words using Google Translate (DeepL is planned too)
+A [maubot](https://github.com/maubot/maubot) to translate words using Google Translate, DeepL or LibreTranslate
```

Comment:
Still no DeepL support, right?

Reply:
I mean, the provider code is there, but I have not tested if it is fully functional, so maybe you are right.

Reply:
Alright, I tested deepl and ran into some issues I was not able to figure out how to fix. So I reverted it's status in the README to planned. Thanks for the catch 👍